###############################################################################
# Copyright (c) 2008, 2018, Oracle and/or its affiliates. All rights reserved.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the "Software"),
# to deal in the Software without restriction, including without limitation
# the rights to use, copy, modify, merge, publish, distribute, sublicense,
# and/or sell copies of the Software, and to permit persons to whom the
# Software is furnished to do so, subject to the following conditions:
#
# The above copyright notice and this permission notice (including the next
# paragraph) shall be included in all copies or substantial portions of the
# Software.
#
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
# DEALINGS IN THE SOFTWARE.
#

$(libXmuu_la_SOURCES) removed from libXmu_la_SOURCES to avoid LD multiply defined errors between libs and mapfile

--- libXmu-1.1.4/src/Makefile.am.orig	2022-10-17 23:09:41.000000000 +0000
+++ libXmu-1.1.4/src/Makefile.am	2022-11-18 21:47:31.752090303 +0000
@@ -9,11 +9,15 @@
 
 AM_CFLAGS = $(CWARNFLAGS) $(XMU_CFLAGS)
 
-libXmu_la_LDFLAGS = -version-number 6:2:0 -no-undefined
+libXmu_la_LDFLAGS = -version-number 4:0:0 -no-undefined
+libXmu_la_LDFLAGS += -Wl,-M,../../../mapfile-Xmu
 libXmuu_la_LDFLAGS = -version-number 1:0:0 -no-undefined
+libXmuu_la_LDFLAGS += -Wl,-M,../../../mapfile-Xmuu
 
-libXmu_la_LIBADD = $(LTLIBOBJS) $(XMU_LIBS)
-libXmuu_la_LIBADD = $(XMUU_LIBS)
+EXTRA_libXmu_la_DEPENDENCIES = libXmuu.la
+
+libXmu_la_LIBADD = $(LTLIBOBJS) $(XMU_LIBS) libXmuu.la -lX11 -lc -lm
+libXmuu_la_LIBADD = $(XMUU_LIBS) -lc
 
 libXmuu_la_SOURCES = \
 	ClientWin.c \
@@ -23,7 +27,8 @@
 	Lower.c
 
 libXmu_la_SOURCES = \
-	$(libXmuu_la_SOURCES) \
+	CmapEquiv.c \
+	VisGamma.c \
 	Xmuint.h \
 	AllCmap.c \
 	Atoms.c \
@@ -79,7 +84,7 @@
 
 lintlib_DATA = $(LINTLIB) $(LINTLIBUU)
 
-$(LINTLIB): $(libXmu_la_SOURCES)
+$(LINTLIB): $(libXmu_la_SOURCES) $(libXmuu_la_SOURCES)
 	$(LINT) -y -oXmu -x $(ALL_LINT_FLAGS) $(libXmu_la_SOURCES)
 
 $(LINTLIBUU): $(libXmuu_la_SOURCES)
